.github/workflows/run_build.yml

```diff
@@ -1,45 +0,0 @@
-# Copyright 2023 DigitalOcean
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-name: Build
-
-on: 
-  push:
-    branches: [main]
-  pull_request:
-    branches: [main]
-
-jobs:
-
-  run-go-build:
-    runs-on: ubuntu-20.04
-
-    steps:
-    - uses: actions/checkout@v2
-
-    - uses: actions/setup-go@v2
-      with:
-        go-version: '1.20'
-
-    - name: Download Ceph repository key
-      run: wget -q 'https://download.ceph.com/keys/release.asc'
-    - name: Install Ceph repository key
-      run: sudo apt-key add release.asc
-    - name: Declare Ceph Nautilus repository
-      run: echo deb https://download.ceph.com/debian-nautilus/ $(lsb_release -sc) main | sudo tee /etc/apt/sources.list.d/ceph.list
```

Comment:
wow, this really still tied to N release?

Reply:
They add naut specifically to just enable support, I don't think it's tied to the release exactly.